library/blob.h

```diff
@@ -129,7 +129,8 @@ inline static int eblob_fdatasync(int fd)
 
 struct eblob_base_ctl {
 	struct eblob_backend	*back;
-	struct list_head	base_entry;
+	struct list_head	base_entry;  // entry of queue of all bases
+	struct list_head	queue_entry;  // entry of queue to sort after close.
```

Comment:
Is naming good enough? Maybe something like `closed_unsorted_base_entry` is more long, but could be more understandable (and no comments required).

Reply:
done